internal/query/gin_map.go

```diff
@@ -2,19 +2,18 @@ package query
 
 import "github.com/gin-gonic/gin"
 
-// QueryNestedMap returns a map for a given query key.
+//revive:disable:exported We want to mimic the gin API.
+
+// ShouldGetQueryNestedMap returns a map from query params.
 // In contrast to QueryMap it handles nesting in query maps like key[foo][bar]=value.
-//
-//revive:disable:exported We want to mimic the gin API
-func QueryNestedMap(c *gin.Context, key string) (dicts map[string]interface{}) {
-	dicts, _ = GetQueryNestedMap(c, key)
-	return
+func ShouldGetQueryNestedMap(c *gin.Context) (dicts map[string]interface{}, err error) {
```

Comment:
As far as I see it, it is returns "one" dictionary - so it should be `dict` (without `s`) 🙈